librad/src/git/types/remote.rs

```diff
@@ -460,6 +460,17 @@ impl Remote<LocalUrl> {
         tracing::debug!("creating temporary remote {}", self.name);
         self.save(repo).unwrap();
         let res = f(self);
+        // remove fetch and push entries so that references do not get removed when
+        // `remote_delete` gets called below
```

Comment:
This seems to address some pathological behaviour, but the comment is only useful in so far as it explains what the following code does, but not why the references would be deleted.